lib/shared/addon/components/cluster-driver/driver-oracleoke/component.js

```diff
@@ -0,0 +1,448 @@
+import Component from '@ember/component'
+import ClusterDriver from 'shared/mixins/cluster-driver';
+import layout from './template';
+import { equal } from '@ember/object/computed'
+
+import {
+  get, set, computed, observer, setProperties
+} from '@ember/object';
+import { inject as service } from '@ember/service';
+
+const regionMap = {
+  'Mumbai':    'ap-mumbai-1',
+  'Seoul':     'ap-seoul-1',
+  'Tokyo':     'ap-tokyo-1',
+  'Toronto':   'ca-toronto-1',
+  'Frankfurt': 'eu-frankfurt-1',
+  'Zurich':    'eu-zurich-1',
+  'Sao Paolo': 'sa-saopaulo-1',
+  'London':    'uk-london-1',
+  'Ashburn':   'us-ashburn-1',
+  'Phoenix':   'us-phoenix-1',
+}
+
+const k8sVersionMap = {
```

Comment:
Is any of this const info available from your API?  We typically pull this kind of stuff from there instead of hardcoding.

You're going to be sort of painting yourselves into a corner when ask this is burnt into versions that users don't necessarily update frequently and you add a new region or don't want to support k8s 1.14.x anymore.

Reply:
I totally agree that we should be pulling in these values dynamically. At some point, I decided to submit that changes as a _subsequent enhancement_ to the driver(s). But your point about users not updating frequently is well taken.

_My preference_ would be to make the enactment as a part this PR. On the other hand, it is arguably an enhancement and there is a desire to have both of the drivers (OKE cluster, and OCI node) on the next release train....

Reply:
We agreed offline that pulling in these values dynamically is desirable, but technically not required and therefore could be submitted as a subsequent _enhancement_.